src/test/suite/fixtures/recorded/hatTokenMap/bringHarpAndPointToEndOfThisAndEndOfWhaleTakeWhale.yml

```diff
@@ -1,7 +1,7 @@
 languageId: plaintext
 command:
   version: 1
-  spokenForm: bring harp and point to end of this and end of whale take whale
+  spokenForm: bring harp and point to end of this and end of whale
```

Comment:
These tests are a bit strange.  The final "take whale" is there to indicate which token we're testing

Reply:
I think you gonna have to explain that further.

Reply:
https://www.cursorless.org/docs/contributing/test-case-recorder/#navigation-map-tests